.github/workflows/build.yml

```diff
@@ -0,0 +1,87 @@
+name: "Build and test Malwarecage"
+on:
+  push:
+    branches:
+    - master
+    - ci/*
```

Comment:
why `ci/*`?

Reply:
I'm not really familiar yet with Github Actions and I don't see any option to manually run workflow for a branch. Might be useful for CI setup branches to run workflows for every push (maybe not?)

Reply:
🤔 Maybe it's not the best idea - removed that.